examples/Main.hs

```diff
@@ -21,18 +21,18 @@ newtype Lit a = Lit Int deriving Functor
 
 -- smart constructor. the :< is pronounced "member":
 -- what this says is that as long as Lit is a member of the type-level
--- list 'fs', we can inject it into an Expr that contains 'fs'
--- if we tried to inject it into an 'Expr [Thing1, Thing2]',
+-- list 'fs', we can inj it into an Expr that contains 'fs'
+-- if we tried to inj it into an 'Expr [Thing1, Thing2]',
 -- we would get an error message that Lit cannot be found in [Thing1, Thing2]
 lit :: (Lit :< fs) => Int -> Expr fs
-lit = In . injectSum . Lit
+lit = In . inj . Lit
```

Comment:
I’d greatly prefer `inject` & `project` to `inj` & `prj`, personally.

Reply:
Fair enough. Fixing